src/sdam/server.ts

```diff
@@ -153,6 +154,8 @@ export class Server extends TypedEventEmitter<ServerEvents> {
       this.clusterTime = clusterTime;
     });
 
+    if (this.loadBalanced) return;
```

Comment:
```suggestion
    // monitoring is disabled in load balancing mode
    if (this.loadBalanced) return;
```
I think it might be useful to call this out here for future maintainers.